src/router.coffee

```diff
@@ -10,6 +11,8 @@ routes = (
     <Route path='dashboard' name='dashboard' handler={Dashboard} />
     <Route path='tasks' name='tasks' handler={Tasks} />
     <Route path='tasks/:id' name='task' handler={SingleTask} />
+    <Route path='reading' name='createReading' handler={CreateReading} />
```

Comment:
I updated it to be /readings, but still like the non-plural form better :).

I also noticed something weird with routes; if I go to /readings/ or /tasks/, I get the invalid state.  Is there a way we can avoid having to add duplicate routes for those?